packages/vats/src/vat-provisioning.js

```diff
@@ -50,7 +51,7 @@ export function buildRootObject(_vatPowers) {
         // Update the notifier when the chainBundle resolves.
         const { notifier, updater } = makeNotifierKit();
         chainBundle.then(clientHome => {
-          updater(harden({ clientHome, clientAddress: address }));
+          updater.updateState(harden({ clientHome, clientAddress: address }));
```

Comment:
Fix LGTM, but I should probably defer to someone more familiar with this code and notifiers. I believe this is the only behavioral difference in this PR.

Reply:
Yep, the only intended behavior change. `updater` comes out of `makeNotifierKit` which just has `updateState`, `fail` and `finish`. The comment says "Update the notifier" so I'm sure it's not meant to be one of the latter two.

FYI @michaelfig who I think wrote this.